packages/fast-development-site-react/src/utilities/error-boundary.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import { toPx } from "@microsoft/fast-jss-utilities";
+import { IDevSiteDesignSystem } from "../../src/components/design-system";
+import manageJss, { ComponentStyles, IJSSManagerProps, IManagedClasses } from "@microsoft/fast-jss-manager-react";
+
+/*tslint:disable:no-empty-interface*/
+export interface IErrorBoundaryProps {}
+/*tslint:disable:no-empty-interface*/
```

Comment:
should this be enable?

Reply:
Just making super sure it was disabled... :)

Updated.